packages/react-components/react-persona/src/components/Persona/usePersona.ts

```diff
@@ -76,6 +80,13 @@ export const usePersona_unstable = (props: PersonaProps, ref: React.Ref<HTMLElem
       })
     : undefined;
 
+  // When there's a PresenceBadge and and Avatar, the presence will be displayed by the Avatar. This means
+  // usePersonaStyles will not add persona's presence className. This ensures it will be added to the Avatar's badge.
```

Comment:
```suggestion
  // When there's a PresenceBadge and an Avatar, the presence will be displayed by the Avatar. This means
  // usePersonaStyles will not add persona's presence className. This ensures it will be added to the Avatar's badge.
```